src/rules/flake8_executable/helpers.rs

```diff
@@ -0,0 +1,68 @@
+use once_cell::sync::Lazy;
+use regex::Regex;
+
+static SHEBANG_REGEX: Lazy<Regex> =
+    Lazy::new(|| Regex::new(r"(?P<spaces>\s*)#!(?P<directive>.*)").unwrap());
+
+#[derive(Debug)]
+pub enum ShebangDirective<'a> {
+    None,
+    Match(usize, usize, usize, &'a str),
```

Comment:
Can we add a comment here to document these fields? (Or use a struct with named fields and comments?) Will make it a bit clearer by disambiguating all the `usize` members.